distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -229,5 +312,35 @@ private String createRegistryEntry(String id) throws Exception {
         return xPath.compile(idPath)
                 .evaluate(xml);
     }
+
+    private void createRegistryStoreEntry(String id) throws Exception {
+        FederationAdminService federationAdminServiceImpl = getServiceManager().getService(
+                FederationAdminService.class);
+        federationAdminServiceImpl.addRegistryEntry(Library.getRegistryNode(id), destinations);
+
+        ValidatableResponse validatableResponse = getRegistryStoreResponse("registry-id", id);
+
+        final String xPathRegistryID =
+                "string(//GetRecordsResponse/SearchResults/RegistryPackage/ExternalIdentifier/@registryObject)";
+        validatableResponse.body(hasXPath(xPathRegistryID, CoreMatchers.is(id)));
+    }
+
+    private ValidatableResponse getRegistryStoreResponse(String attr, String value) {
```

Comment:
Might want to consider renaming this method since it is not going through the registry store.